### ISSUE TITLE
Links to Dive into Python in the testing documentation are dead

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7668]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Alex reported|
|-|-|
|Trac ID|trac#7668|
|Type|defect|
|Created|2014-10-05 20:40:35Z|
|Branch|https://github.com/twisted/twisted/tree/doc-dev-dive-link-7668|

https://twistedmatrix.com/documents/current/core/development/policy/test-standard.html#links

Mark Pilgrim took the website down, and now these links are dead. They should either point to someone who's rehosting the book (there are a few I believe), or deleted.

Attachments:

* [fixing-broken-doc-link.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/9eb/9eb4f40e2b21953699e31c144e39eb2cf963a1e6/1c4355036eaec5907a7714c831b71f5e34a3ecc5.patch) (994 bytes) - added by cwron on 2014-10-08 19:09:19Z - Link fixes

<details><summary>Searchable metadata</summary>

```
trac-id__7668 7668
type__defect defect
reporter__Alex Alex
priority__normal normal
milestone__ 
branch__branches_doc_dev_dive_link_7668 branches/doc-dev-dive-link-7668
branch_author__adiroiban adiroiban
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1412541635551713 1412541635551713
changetime__1447938260008946 1447938260008946
version__None None
owner__thijs thijs

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban commented|
|-|-|

(In [46262]) Branching to doc-dev-dive-link-7668.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @thijstriemstra|
|-|-|

Many thanks for your contributions.

Changes looks good.

Will wait for test to run and if all is ok (as it should be) I will merge.

Thanks again!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban set status to `closed`|
|-|-|

(In [46286]) Merge doc-dev-dive-link-7668: Fix links to Dive into Python.

Author: thijs
Reviewer: adiroiban
Fixes: [#7668](https://github.com/twisted/twisted/issues/7668)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra commented|
|-|-|

Thanks adiroiban. I merely put someone else's patch up for review, but I'm credited as the author which should be cwron. Make sure to double-check that before merging :)
